### ISSUE TITLE
Possible discrepancy in updating V(s)

### ISSUE THREAD

**Issue by Erfi** [hari-sikchi/DVL#2]
When comparing the implementation of the V(s) update to the equation (14) I am confused by the `-` sign in front of `(1-alpha)*diff`. Shouldn't this be a `+`?

![14](https://github.com/hari-sikchi/DVL/assets/3267547/58ce6b85-3956-44ae-827e-2ae872464672)

```
def chi_square_loss(diff, alpha, args=None):
    loss = alpha*jnp.maximum(diff+diff**2/4,0) - (1-alpha)*diff
    return loss
```

**Comment by hari-sikchi** [comment]
I think the difference is that the loss in implementation is written in terms of diff=Q-V where Q is detached, so it's not updated. -(1-alpha)(Q-V) is basically updating (1-alpha)V.

**Comment by Erfi** [comment]
It makes sense regarding gradient flow, but is there any advantage in doing it that way? cause essentially we are subtracting (1-alpha)Q from the loss which changes the proportion of the values between the first and second term of Eq.14. Perhaps adjusting the alpha then can compensate for it 🤔
